noir-projects/noir-contracts/contracts/pending_note_hashes_contract/src/main.nr

```diff
@@ -115,7 +115,7 @@ contract PendingNoteHashes {
         note.randomness = 2;
 
         // Insert note
-        owner_balance.insert(&mut note, true, outgoing_viewer_ovpk_m, owner_ivpk_m);
+        owner_balance.insert(&mut note, outgoing_viewer_ovpk_m, owner_ivpk_m);
```

Comment:
The broadcast param has been nuked in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/6741)